.github/workflows/build.yml

```diff
@@ -133,6 +133,7 @@ jobs:
           cp canary-release-changeset.md .changeset
           yarn changeset pre enter ${date}
           yarn changeset version
+          yarn changeset pre exit
```

Comment:
this ends the prerelease phase in which we created the version with the date tag. Then we want to do a normal publish with the tag `canary` which is not possible in prerelease phase. So we exit the phase. This should be already on main and I guess it is shown only because I did not merge main.

Reply:
k.